line/line-sdk-android
# Replace Sample App

## Description
- Replace the existing smaple app.
  - The new version of the sample app is completely written in kotlin and uses Jetpack Compose to design the UI.

- In addition, some SDK's dependency versions were also upgraded.
- Update Java version to 11 in build CI.

## How to verify?
- Run these two tasks to check if they build successfully: 
  - `:app:assembleDebug`
  - `:line-sdk:assembleDebug`

- Run the new Sample App to an emulator/physical device.

## Screenshot

<table>
<tr>
<td><img src="https://user-images.githubusercontent.com/47718989/191452449-69cb8349-f556-461d-9661-047de2aee9b9.jpg"></td>
<td><img src="https://user-images.githubusercontent.com/47718989/191461253-6f1e5668-036a-4ccd-a31f-01bccd453c7c.jpg"></td>
</tr>
</table>
